packages/client/src/components/ClientApp.svelte

```diff
@@ -93,6 +94,16 @@
 
   // Load app config
   onMount(async () => {
+    if ("serviceWorker" in navigator) {
```

Comment:
Should this not be part of the `InstallPrompt` in order to encapsulate this logic?